src/api/alerts.ts

```diff
@@ -132,10 +132,11 @@ const getNotificationSubscriptionForUser = async (
     const data = await supabaseRetry(
         () =>
             supabaseClient
-                .from<AlertSubscriptionQuery>(TABLES.ALERT_SUBSCRIPTIONS)
+                .from(TABLES.ALERT_SUBSCRIPTIONS)
                 .select(`id, catalog_prefix, email`)
                 .eq('catalog_prefix', prefix)
-                .eq('email', email),
+                .eq('email', email)
+                .returns<AlertSubscriptionQuery[]>(),
```

Comment:
This will be one of the patterns of changes where we manually set the return of the queries for now since we are not using the typing when creating client.